ftd/src/interpreter/main.rs

```diff
@@ -1349,6 +1349,85 @@ impl Document {
         Ok(None)
     }
 
+    // Returns (response, content-type, status, headers)
+    #[allow(clippy::type_complexity)]
+    pub fn get_response(
+        &self,
+    ) -> ftd::interpreter::Result<Option<(String, String, u16, fastn_resolved::Map<String>)>> {
+        use ftd::interpreter::{PropertyValueExt, ValueExt};
+
+        let tdoc = self.tdoc();
+
+        for v in self.get_instructions("ftd#response") {
+            if !is_visible(self, &v)? {
+                continue;
+            }
+
+            let response = match v.get_interpreter_value_of_argument("response", &tdoc)? {
+                Some(val) => remove_escape(val.string(self.name.as_str(), v.line_number)?.as_str()),
+                None => {
+                    return ftd::interpreter::utils::e2(
+                        "response not found in ftd.response",
+                        self.name.as_str(),
+                        v.line_number,
+                    );
+                }
+            };
+
+            let content_type = match v.get_interpreter_value_of_argument("content-type", &tdoc)? {
+                Some(val) => val.string(self.name.as_str(), v.line_number)?,
+                None => {
+                    return ftd::interpreter::utils::e2(
+                        "content-type not found in ftd.response",
+                        self.name.as_str(),
+                        v.line_number,
+                    );
+                }
+            };
+
+            let status_code = match v.get_interpreter_value_of_argument("status-code", &tdoc)? {
+                Some(val) => val.integer(self.name.as_str(), v.line_number)?,
+                None => 200,
+            };
+
+            if !(100..1000).contains(&status_code) {
```

Comment:
Instead of explicit logic here, let's use `actix_web::http::StatusCode::from_u16`. We will have to do both checks, check that it is less than u16::MAX, if so cast it as u16, and then use from_u16 to see if its valid.